managed-tests/shared/src/test/scala/zio/managed/HasNoScopeSpec.scala

```diff
@@ -0,0 +1,56 @@
+package zio.managed
```

Comment:
I feel like this is the wrong place with this suite. Any reason it's not under the `core-tests` folder?

Reply:
I just took a look where the other tests that use macros where. I can move it

Reply:
Yes let's please move it. I'm not even sure why the other macro tests are under there but we should probably move those too